tests/validation/tests/v3_api/test_workload.py

```diff
@@ -27,14 +26,13 @@ def test_wl_sidekick():
                                         containers=con,
                                         namespaceId=ns.id)
     validate_workload(p_client, workload, "deployment", ns.name)
+
     side_con = {"name": "test2",
-                "image": "ubuntu",
-                "stdin": True,
-                "tty": True}
+                "image": TEST_IMAGE_NGINX}
```

Comment:
Why should this code, be removed? 
We would like to keep this for the linux environment testing.

"stdin": True,	
"tty": True}

Reply:
Updated